# Maximum likelihood parameters
measurement_model: los_nlos
los_mean_low_var: 2.51
los_sd_low_var: 1.5
los_mean_high_var: 2.51
los_sd_high_var: 1.5
var_cutoff: 0.20

num_mixtures: 2

nlos_mean0: -5.53
nlos_sd0: 1.0
nlos_proportion0: 0.25

nlos_mean1: -5.25
nlos_sd1: 11.0
nlos_proportion1: 0.75

nlos_simple_mean: -5.53
nlos_simple_sd: 4.0
